vite.config.ts

```diff
@@ -5,4 +5,13 @@ import loadVersion from 'vite-plugin-package-version';
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react(), loadVersion()],
+  resolve: {
+    // this is only applicable when npm-linking the utah-design-package
+    dedupe: ['firebase', '@arcgis/core'],
+  },
+  build: {
+    rollupOptions: {
+      external: [/@arcgis\/core\/.*/],
```

Comment:
**question**: is required since it's referenced in a dependency even though it's not directly imported?

Reply:
The build was failing with an error saying that it couldn't resolve @arcgis/core modules. But then this breaks the build. I fixed it by removing the dedupe config. But there's a TON of @arcgis/core modules in the bundle even though we don't use them in the app. I wonder if it's because kitchen sink defines it as a dep rather than peer dep? I don't know. I'm not too worried about the bundle size for this app.